lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -52,14 +53,14 @@ export class AuthorizationCodeClient extends BaseClient {
      * authorization_code_grant
      * @param request
      */
-    async acquireToken(request: AuthorizationCodeRequest, cachedNonce?: string, cachedState?: string): Promise<AuthenticationResult> {
+    async acquireToken(request: AuthorizationCodeRequest, telemetryManager?: TelemetryManager, cachedNonce?: string, cachedState?: string): Promise<AuthenticationResult> {
```

Comment:
Instead of passing in TelemetryManager into every acquireToken call, would it might make more sense to include as when you create the AuthorizationCodeClient (by passing into the ClientConfiguration). This applies to all the other objects that extend BaseClient. 

You could also potentially move 

```
if (telemetryManager) {
            headers = telemetryManager.addTelemetryHeaders(headers);
        }
```
to executePostToTokenEndpoint in BaseClient, and remove the duplication